test/types/schema.test.ts

```diff
@@ -408,7 +408,7 @@ export function autoTypedSchema() {
     array1: string[];
     array2: any[];
     array3: any[];
-    array4: any[];
+    // array4: any[];
```

Comment:
what happened to this? shouldnt this be removed if it is going to be unused?

Reply:
This was mistakenly commented out for testing purposes, looks like my fix is incomplete :( thanks for pointing this out.